docs/proposals/PROPOSAL_C.md

```diff
@@ -321,3 +267,66 @@ Gives response:
+1. As a tool writer, I want to identify whether a registry supports reference types or not.
+  - Yes: use `GET /v3/`. Catalog listing is not prescribed in this proposal.
+1. As a tool writer, I would like the option to perform a server side blob mount when copying a large artifact between repositories.
+  - Yes: If a large blob is pushed to the registry and then referenced by a Node manifest, a server side mount should be possible.
+1. As a tool writer, I want to be able to include reference types within the Image Layout filesystem format.
+  - Yes: The existing image layout will be unaffected.
+
+#### Content Management
+
+1. As a user, I want to store one or more artifacts in a registry with a reference to another related artifact.
+  - Yes: Node manifests can group one or more artifacts with a reference to another manifest.
+1. As a user, when I delete an artifact, I want the option to delete one or more artifacts referencing it.
+  - Partial: use `GET /v3/<name>/referrers/<reference>` to get a list of referring artifacts, use the client to delete specific ones, then push a new Node manifest containing the updated object list. To delete the whole manifest, use `DELETE /v3/<name>/manifests/<reference>`.
+1. As a user, I want to push an artifact that references another artifact that doesn't exist on the registry yet.
+  - Yes: This is should be possible if registries don't perform a check that the digest exists.
+1. As a user, I want to move an artifact and one or more artifacts referencing it from one registry to another.
+  - Partial: The client is responsible for selection of the artifacts and pushing to the target registries, including checks if the registry supports node manifests.
+1. As a registry operator, I want to help users understand how they can manage the lifecycle of their artifacts.
+  - Yes: Node manifests allow for lifecycle policies to be applied to the list of objects.
+1. As a registry operator, I want to allow users to "lock" the tags to their artifacts.
+  - Yes: Collections of different types of artifacts may be referenced by a long lived tag.
+1. As an artifact producer, I want to update an existing artifact with a newer artifact.
+  - Partial: use `GET /v3/<name>/manifests/<reference>` to fetch a node manifest, then update the objects list with a new artifact.
+1. As an artifact producer, I want to push multiple artifacts concurrently (possibly from separate systems) without encountering race conditions or other errors.
+  - No: If two manifests with the same digest are pushed to the same registry path, a race condition will occur. However, this problem exists in current registries.
+1. As an artifact author, I want to document other artifacts in one or more registries that my artifact requires and/or provides.
+  - Partial: Use the description to indicate the reference is a "requires" or "provides" relationship.
```

Comment:
This one depends on whether registries are validating that referred artifacts exist. Cross repository or even registry descriptor references won't work with most of our proposals if there's validation.